backend/Watchdog.Core/Watchdog.Core.BLL/MappingProfiles/ApplicationProfile.cs

```diff
@@ -20,9 +20,15 @@ public ApplicationProfile()
                     s => s.AlertCategory == Common.DTO.Application.AlertSettings.AlertCategory.Special 
                     ? s.SpecialAlertSetting 
                     : null));
+            CreateMap<AlertSetting, AlertSettingsDto>();
```

Comment:
Why do you have these mapping in two files?

Reply:
Yesterday I tried different mapping options. It was the right option, but I decided to put it in a separate file. Deleted this file)